addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/BrowserBasedAuthenticationMethodType.java

```diff
@@ -645,17 +645,19 @@ public void importData(Configuration config, AuthenticationMethod authMethod)
             throw new ConfigurationException(e);
         }
         try {
-            method.setBrowserId(config.getString(CONTEXT_CONFIG_AUTH_BROWSER_BROWSERID));
+            method.setBrowserId(
+                    config.getString(CONTEXT_CONFIG_AUTH_BROWSER_BROWSERID, DEFAULT_BROWSER_ID));
         } catch (Exception e) {
             throw new ConfigurationException(e);
         }
         try {
-            method.setLoginPageWait(config.getInt(CONTEXT_CONFIG_AUTH_BROWSER_LOGINPAGEWAIT));
+            method.setLoginPageWait(
+                    config.getInt(CONTEXT_CONFIG_AUTH_BROWSER_LOGINPAGEWAIT, DEFAULT_PAGE_WAIT));
```

Comment:
These should be reverted, they existed from the beginning.

Reply:
Yes, old exports have those fields since they were there from the beginning. If you are asking about the case where the data exported is tampered with that's not a problem we should handle, the data is not to be manually changed.